src/test/ast/inline-document-descriptor_test.ts

```diff
@@ -14,41 +14,40 @@
 
 import {assert} from 'chai';
 
-import {LocationOffset, correctSourceLocation} from '../../ast/ast';
-import {SourceLocation} from '../../elements-format';
+import {LocationOffset, Position, SourceRange, correctPosition, correctSourceRange} from '../../ast/ast';
 
 
-suite('correctSourceLocation', function() {
+suite('correctSourceRange', function() {
   test('handles undefined gracefully', function() {
-    const zeroSourceLocation: SourceLocation = {line: 0, column: 0};
+    const zeroPosition = {line: 0, column: 0};
+    const zeroSourceRange:
+        SourceRange = {file: 'foo', start: zeroPosition, end: zeroPosition};
     const zeroLocationOffset: LocationOffset = {line: 0, col: 0};
 
-    assert.equal(undefined, correctSourceLocation(undefined, undefined));
-    assert.equal(
-        undefined, correctSourceLocation(undefined, zeroLocationOffset));
+    assert.equal(undefined, correctSourceRange(undefined, undefined));
+    assert.equal(undefined, correctSourceRange(undefined, zeroLocationOffset));
     assert.deepEqual(
-        zeroSourceLocation,
-        correctSourceLocation(zeroSourceLocation, undefined));
+        zeroSourceRange, correctSourceRange(zeroSourceRange, undefined));
   });
   test('handles source locations on the first line', function() {
     assert.deepEqual(
         {line: 1, column: 2},
-        correctSourceLocation({line: 0, column: 1}, {line: 1, col: 1}));
+        correctPosition({line: 0, column: 1}, {line: 1, col: 1}));
   });
   test(
       'does not change column offsets for ' +
           'source locations after the first',
       function() {
         assert.deepEqual(
             {line: 2, column: 1},
-            correctSourceLocation({line: 1, column: 1}, {line: 1, col: 1}));
+            correctPosition({line: 1, column: 1}, {line: 1, col: 1}));
       });
 
   test('does not modify its input', function() {
-    const input: SourceLocation = {line: 5, column: 5};
+    const input: Position = {line: 5, column: 5};
     const offset: LocationOffset = {line: 5, col: 5};
-    const expected: SourceLocation = {line: 10, column: 5};
-    assert.deepEqual(expected, correctSourceLocation(input, offset));
+    const expected: Position = {line: 10, column: 5};
```

Comment:
It's initially counterintuitive, but no! We only should correct the column offset on the *first line* of the inline document. Otherwise the column offsets are correct.